cli/cmd/plugin/ui/web/tanzu-ui/src/views/management-cluster/aws/AwsClusterSettingsStep.tsx

```diff
@@ -0,0 +1,186 @@
+// React imports
+import React, { useContext, useState, useEffect } from 'react';
+
+// Library imports
+import { ClarityIcons, blockIcon, blocksGroupIcon, clusterIcon } from '@cds/core/icon';
+import { useForm } from 'react-hook-form';
+import { CdsIcon } from '@cds/react/icon';
+import { CdsButton } from '@cds/react/button';
+
+// App imports
+import { INPUT_CHANGE } from '../../../state-management/actions/Form.actions';
+import { StepProps } from '../../../shared/components/wizard/Wizard';
+import { STORE_SECTION_FORM } from '../../../state-management/reducers/Form.reducer';
+import {
+    NodeProfile,
+    NodeInstanceType,
+    nodeInstanceTypeValidation,
+} from '../../../shared/components/FormInputComponents/NodeProfile/NodeProfile';
+import { ClusterName, clusterNameValidation } from '../../../shared/components/FormInputComponents/ClusterName/ClusterName';
+import { AwsStore } from '../../../state-management/stores/Store.aws';
+import { yupResolver } from '@hookform/resolvers/yup/dist/yup';
+import * as yup from 'yup';
+import { AwsService, AWSVirtualMachine } from '../../../swagger-api';
+import RetrieveOSImages from '../../../shared/components/FormInputComponents/RetrieveOSImages/RetrieveOSImages';
+
+ClarityIcons.addIcons(blockIcon, blocksGroupIcon, clusterIcon);
+
+type AWS_CLUSTER_SETTING_STEP_FIELDS = 'NODE_PROFILE' | 'IMAGE_INFO' | 'CLUSTER_NAME';
+
+interface MCSettings extends StepProps {
+    message?: string;
+    deploy: () => void;
+    defaultData?: { [key: string]: any };
+}
+
+const nodeInstanceTypes: NodeInstanceType[] = [
+    {
+        id: 'SINGLE_NODE',
```

Comment:
please leverage the AWS_NODE_PROFILE_NAMES enum defined in AwsManagementClusterBasic.constants.tsx file. If you cannot find this file, please synch with top of tree and it should show up.